tests/mypy/outputs/1.0.1/mypy-plugin_ini/plugin_strict_fields.py

```diff
@@ -1,56 +1,56 @@
-from pydantic import BaseModel, Field

```

Comment:
I'm not sure what is actually changed in this file. Everything looks identical. It was "updated" after I ran `make test-mypy-update-all`.